### PR TITLE
TC election process

### PR DESCRIPTION
**Pull request by DavidPHirsch**
Adding TC election process
FYI @toddbaert

### CODE REVIEW

**Review by beeme1mr — COMMENTED** [review]
What's the GC involvement in this process (if any)?

**Comment by toddbaert** [comment]
@DavidPHirsch sorry for missing this until now!

This _sounds_ good to me. How did you come up with this? Does it draw inspiration from any existing models?

**Comment by toddbaert** [comment]
The above sounds good to me, though I wouldn't consider myself a governance expert.

@DavidPHirsch I've noticed there's no term length specified. This is intentional, yes? Can you clarify, and perhaps explicitly document that?

**Review by toddbaert — APPROVED** [review]

**Review by thomaspoignant — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]

File: tech-committee-charter.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ TC membership is developed through leadership in the project and was bootstrappe
 Changes to TC membership should be posted in the agenda document, and may be suggested as any other agenda item.
 
 ### No Over-Representation
-
+No more than one-fourth (25%) of the TC members may be affiliated with the same employer (in the event of confusion or concern, the OpenFeature TC will defer to the CNCF definition of "same employer"). If removal or resignation of a TC member, or a change of employment by a TC member, creates a situation where more than one-fourth of the TC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more TC members affiliated with the over-represented employer(s).
```
Comment (AlexsJones) [inline tech-committee-charter.md:21]: Do you have a link to that definition?

Comment (beeme1mr) [reply tech-committee-charter.md:21]: I wasn't able to find a formal definition on the CNCF site. This is something we should address before merging. If the CNCF no longer has a published definition, we should either ask them to republish it or defined it ourselves.

I also noticed this was the same language used in the [OTel TC charter](https://github.com/open-telemetry/community/blob/main/tech-committee-charter.md).

Comment (toddbaert) [inline tech-committee-charter.md:21]:
```suggestion
No more than one-third (33%) of the TC members may be affiliated with the same employer (in the event of confusion or concern, the OpenFeature TC will defer to the CNCF definition of "same employer"). If removal or resignation of a TC member, or a change of employment by a TC member, creates a situation where more than one-fourth of the TC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more TC members affiliated with the over-represented employer(s).
```

**Comment by DavidPHirsch** [comment]
@AloisReitbauer @AlexsJones @justinabrahms @moredip @dabeeeenster 
Please have a look

File: tech-committee-charter.md [hunk 2]
```diff
@@ -65,6 +68,17 @@ In such cases, TC members will inform the the wider TC of the perceived conflict
 
 ### Election of TC Members
 
+New TC members can be nominated by any Member in Standing as defined in the [OpenFeature Governance Charter](https://github.com/open-feature/community/blob/main/governance-charter.md), and must be sponsored by one of the existing TC members to bring it to a vote. A candidate can be elected to the TC by the super-majority vote (greater than two thirds) of the existing TC members. The voting by the TC members is anonymous. A multiple-candidate method, such as [Condorcet](http://en.wikipedia.org/wiki/Condorcet_method) or [Single Transferable Vote](http://en.wikipedia.org/wiki/Single_transferable_vote), may be used to conduct the vote, by phrasing the selections as:
```
Comment (AloisReitbauer) [inline tech-committee-charter.md:71]: Do we really need a complex voting method. Majority from the GC should be fine.

**Review by AloisReitbauer — CHANGES_REQUESTED** [review]

**Review by justinabrahms — APPROVED** [review]

File: tech-committee-charter.md (hunk 2 above)
Comment (AloisReitbauer) [reply tech-committee-charter.md:71]: Can we please simplify the voting as discussed on the last GC call:

New TC members must be nominated by existing TC members. GC has a veto for new members. 

The TC servers as the technical arm of the GC. The GC is already voted by the wider community, so no need to make everything a community vote here.

File: tech-committee-charter.md (hunk 1 above)
Comment (AloisReitbauer) [reply tech-committee-charter.md]: Same employer might be misleading in some cases (e.g. contract work which is pretty common in OSS). I would rephrase to "representing the same legal entity/project"

Comment (AloisReitbauer) [reply tech-committee-charter.md]: OK, this is about the TC. As the TC has no formal "power", I am not sure how strict we need to be here.

**Review by DavidPHirsch — COMMENTED** [review]
adding changes

**Comment by DavidPHirsch** [comment]
added the changes here https://github.com/open-feature/community/pull/183 the new permissions do not let me edit this PR

**Comment by beeme1mr** [comment]
You should have access again. Sorry about that.

File: tech-committee-charter.md (hunk 1 above)
Comment (toddbaert) [reply tech-committee-charter.md]: I think it's not a bad idea anyway.

**Review by toddbaert — APPROVED** [review]

File: tech-committee-charter.md [hunk 3]
```diff
@@ -65,6 +68,12 @@ In such cases, TC members will inform the the wider TC of the perceived conflict
...
-
+New TC members can be nominated by an existing TC Member. A candidate can be elected to the TC by the super-majority vote (greater than two thirds) of the existing TC members. The GC will have a veto for the new members.
 
 ### Election of TC Chair
+
+The TC Chair will rotate amongst TC members. The responsibilities of TC Chair include: building an agenda for TC meetings, encouraging participation of other members and facilitating the decisions among TCs. The TC Chairperson should hold their role for 3 months; there are no limits on the number of terms a TC Chairperson may serve.
+
+
+
+
```
Comment (toddbaert) [inline tech-committee-charter.md:79]:
```suggestion

```
